### PR TITLE
Added username password label

### PR DESCRIPTION
**Pull request by Jagrutivyas**

### CODE REVIEW

**Review by rahulkpareek — APPROVED** [review]
LGTM!!!

File: login.py [hunk 1]
```diff
@@ -1,23 +1,46 @@
 from tkinter import *
-import Handlers
+import tkinter.messagebox
+
+#import Handlers
```
Comment (rahulkpareek) [inline login.py:4]: this comment can be deleted

File: login.py [hunk 2]
```diff
@@ -1,23 +1,46 @@
...
+      username1 = usernamevalue.get()
+      password1 = passwordvalue.get()
+
+      if len(username1) <= 0:
+          tkinter.messagebox.showwarning( "Message", "User Name can not be empty")
+      elif len(password1) <= 0:
+          tkinter.messagebox.showwarning( "Message", "password can not be empty")
+      else :
+          tkinter.messagebox.showwarning( "Message", "Not implemented yet..")
+
```
Comment (rahulpareek-room) [inline login.py:19]: Better if moved to a different module. This is our validation logic and shouldn't be mixed up with the UI logic.

File: login.py [hunk 3]
```diff
@@ -1,23 +1,46 @@
...
 #name to be changed with newer version release
 wnd.title("PasswordVault 1.0")
-wnd.geometry('500x400')
+wnd.geometry('400x400')
+
+#photo
+photo = PhotoImage(file="Logo.png")
+image_label = Label(image=photo)
```
Comment (rahulkpareek) [inline login.py:26]: Photo used just once.  Can just use one statement like image_label=Label(image=Photoimage(.....))

File: login.py [hunk 4]
```diff
@@ -1,23 +1,46 @@
...
+image_label.grid(row=0,column=0)
+
 
-#Username Entry
-L1 = Label(wnd, text="UserName").grid(row=1, column=0)
-E1 = Entry(wnd, bd =5).grid(row=1, column=11)
+#Username and Password Labels
+username = Label(wnd, text="UserName").grid(row=1, column=0)
```
Comment (rahulkpareek) [inline login.py:31]: name it username_label

File: login.py [hunk 5]
```diff
@@ -1,23 +1,46 @@
...
+password = Label(wnd, text="Password").grid(row=9, column=0)
```
Comment (rahulkpareek) [inline login.py:32]: name it password_label

**Review by rahulkpareek — COMMENTED** [review]
Review done!!! Need some changes in the code.

**Pull request merged**